### DISCUSSION TITLE
Alitude view

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#1153] [General]
A new beginner question ;)
How can I display the heights of all tours in the same pattern, whether the changes in the altitude are only 10m or 300m?
I know there is a setting for max and min values, but that only works for me with the currently displayed tour and not for all tours. And only if I change it currently.

**Comment by FJBDev** [comment]
You don't give much details so I assume you are talking about the Tour Chart.

Did you look at the Min/Max slideout ? Documentation [here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/tour-chart#graph-min-max)

![image](https://github.com/mytourbook/mytourbook/assets/25235665/860f8989-8f8d-45da-9d4f-3baa35849ec9)

**Reply by ovahead** [reply]
That's exactly what i meant!
I already set the minimum elevation of 0 and a max of 300. But no matter what I set there, it is only relevant for the currently displayed tour and not all...

I just don't want a tour that only has an altitude change of e.g. 10m to be displayed inflated ;)

**Reply by FJBDev** [reply]
Can you add a screenshot of what you are seeing ?
Sometimes a picture is worth a 1000 words :-)

**Reply by ovahead** [reply]
![1](https://github.com/mytourbook/mytourbook/assets/112829463/b8719086-9326-4535-a0f1-627946a1c28b)

If the max value for the altitude is already set to 300 it will only show up to 90m in this case...
Only when I set the maximum again to 300m (checkmark out and back in) it show me from 0 up to 400m:

![2](https://github.com/mytourbook/mytourbook/assets/112829463/0594d253-3b4c-4d44-b10e-4014e30a760a)

And as I wrote, only for the currently selected/displayed tour, not for all...

**Reply by FJBDev** [reply]
Can you show your settings for the Graph Min/Max values slideout ?

**Reply by ovahead** [reply]
![Slideout](https://github.com/mytourbook/mytourbook/assets/112829463/a4e87959-2f41-407f-a660-ffd5cd507625)

What I also noticed in the altitude adjustment (context menu) the tour is displayed correctly as set

![Hoehenasnpassung](https://github.com/mytourbook/mytourbook/assets/112829463/e96e807b-c706-4f45-a2fe-e9b6ee384e45)

**Reply by FJBDev** [reply]
Thanks.
I've tried to reproduce the issue you mention but it seems to be working fine
I've set the same min/max values for the elevation than you (0m to 300m) and it works fine:
![image](https://github.com/mytourbook/mytourbook/assets/25235665/6a1c9322-f5c9-41e4-a1a4-0c2c1c868c40)

Tour with max elevation < 300m ✔️ 
![image](https://github.com/mytourbook/mytourbook/assets/25235665/ffb70513-a6d0-4a75-86fe-1d2fa6e4d31f)

Tour with max elevation < 1900m ✔️
![image](https://github.com/mytourbook/mytourbook/assets/25235665/93dec1b3-d0da-4247-b294-87bf7d861f2e)

**Reply by ovahead** [reply]
Then it is normal for a tour of a maximum of e.g. 75 m that the graphs only show up to a maximum of 90 m, although the maximum value of 300 is set?
However, this changes with the currently displayed tour if you remove the tick from the maximum value and set it again to 300. Then it will be shown up to 400 m... And in Adjust Elevation (context menu) the graph is up 320 m from the beginning.

Now it's may be not a bug, but I think it's in any case an odd behavior ;)
Shouldn't all graphs then be displayed the same for all tours up to a maximum of 300 in all views (e.g. Tour Chart, Adjust Elevation), even if the maximim altitude of the tour is only 75 or 90 m is?

**Comment by wolfgang-ch** [comment]
The maximum value depends also on the chart pixel height and the vertical pixel grid distance

![image](https://github.com/mytourbook/mytourbook/assets/1283445/1c5be957-4410-4536-9e9a-4057822ab9c5)
